modules/afsmtp/afsmtp.c

```diff
@@ -134,6 +135,7 @@ afsmtp_dd_set_from(LogDriver *d, const gchar *phrase, const gchar *mbox)
 
   g_free(self->mail_from->phrase);
   g_free(self->mail_from->address);
+
   self->mail_from->phrase = afsmtp_wash_string(g_strdup(phrase));
```

Comment:
Take care of functions that return pointer to dynamically allocated memory. (E.g.: g_strdup, wash_string) These gchar* strings should be freed when no longer needed. See glib documentation: [here](https://developer.gnome.org)